packages/core/src/Snackbar/StackedSnackbarProvider/StackedSnackbarProvider.js

```diff
@@ -0,0 +1,143 @@
+import React, { forwardRef } from "react";
+import { makeStyles } from "@material-ui/core";
+import PropTypes from "prop-types";
+import { SnackbarProvider, SnackbarContent, useSnackbar } from "notistack";
+
+import HvSnackbarContentWrapper from "../SnackbarContentWrapper";
+
+// We override notistack hook to be able to customize the snackbar that should be called.
+const useHvStackedSnackbar = () => {
+  const { enqueueSnackbar: enqueueNotistackSnackbar, ...others } = useSnackbar();
+  const enqueueSnackbar = (message, options) => {
+    const { id, variant = "success", snackbarContentProps, ...otherOptions } = options;
+    enqueueNotistackSnackbar(
+      JSON.stringify({ message, variant, id, snackbarContentProps }),
```

Comment:
Why not create here the `HvNotistackSnackMessage` instead of stringifying? That will block the ability to pass a node as label.

Notistack's SnackbarMessage can be string or React.ReactNode.